package.json

```diff
@@ -203,6 +205,16 @@
                 "icon": "$(debug-restart)",
                 "category": "Java"
             },
+            {
+                "command": "java.test.editor.run",
+                "title": "%contributes.commands.java.test.editor.run.title%",
+                "category": "Java"
+            },
+            {
+                "command": "java.test.editor.debug",
+                "title": "%contributes.commands.java.test.editor.debug.title%",
+                "category": "Java"
+            },
```

Comment:
You only define a command in this PR, why not to expose it to the editor context menu either?

Reply:
Per offline discussion. We will expose these commands into editor context menu (through new submenu contribution point) and editor title in the next release (0.26.0).